onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -246,6 +258,50 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
           AttentionTypeAndShapeInference(ctx, past_input_index);
         }));
 
+constexpr const char* CrossAttention_ver1_doc = R"DOC(
+Multi-Head Cross Attention. Bias from input projection is included.
+
+The key padding mask is optional. When its shape is (batch_size, kv_sequence_length), value 0
+means padding or 1 otherwise. When key has right-side padding, its shape could be (batch_size): it is actual length of
+each key sequence excluding paddings.
+)DOC";
+
+ONNX_MS_OPERATOR_SET_SCHEMA(
+    CrossAttention, 1,
+    OpSchema()
+        .SetDoc(CrossAttention_ver1_doc)
+        .Attr("num_heads", "Number of attention heads", AttributeProto::INT)
+        .Input(0,
+               "query",
+               "Query with shape (batch_size, sequence_length, hidden_size) when weights is not available.",
+               "T")
+        .Input(1,
+               "key",
+               "Key with shape (batch_size, kv_sequence_length, hidden_size)",
+               "T")
+        .Input(2,
+               "value",
+               "Value with shape (batch_size, kv_sequence_length, v_hidden_size)",
+               "T")
+        .Input(3,
```

Comment:
optional?

Reply:
Packed kv will be supported later. Will update the interface accordingly at that time.